### DISCUSSION TITLE
basic-types/intro

### DISCUSSION THREAD

**Discussion by giscus** [sunface/rust-by-practice#368] [Book Comments]
# basic-types/intro

Learning Rust By Practice, narrowing the gap between beginner and skilled-dev with challenging examples, exercises and projects.

https://zh.practice.rs/basic-types/intro.html

**Comment by Classpi** [comment]
没有评论？🥹🥹🥹

**Comment by jieqiyue** [comment]
承包空评  你樾哥学rust第一天

**Comment by saadaltafofficial** [comment]
day 1
